### DISCUSSION TITLE
how to reply to a specific post?

### DISCUSSION THREAD

**Discussion by geschaenk** [myConsciousness/atproto.dart#1141] [Q&A] [answered]
Hi I wonder how to reply to a specific post via bsky cli.
Kind regards

**Comment by myConsciousness** [comment]
Hi @geschaenk !

bluesky_cli does not currently support replies. This is because I didn't understand the demand for advanced post from bluesky_cli when I implemented the `post` command.

But anyway it's possible to implement this, so I'll let you know when it's released :)

**Reply by myConsciousness** [reply, marked as answer]
- https://github.com/myConsciousness/atproto.dart/issues/1144

**Reply by geschaenk** [reply]
I'd really like this possibility, thank you very much. 👍👍

**Reply by myConsciousness** [reply]
@geschaenk 

I released `bluesky_cli v0.4.2`! You can reply as follows now:

```bash
bsky post --text="replying..." --reply=at://did:plc:iijrtk7ocored6zuziwmqq3c/app.bsky.feed.post/3khingrezew2v
```

**Comment by geschaenk** [comment]
Great work! Thank you so much!
